### PR TITLE
Fix #2118: AdminPinActivity for tablet in both portrait and landscape

### LINKED ISSUE #2118: [Tablet] Highfi AdminPinActivity
Currently `AdminPinActivity` screen does not have dedicated tablet layout design in develop.
Implement the `AdminPinActivity` for landscape and portrait mode for tablet devices.
Mock Link: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/6876c089-6ee6-48d4-85bb-880be3b5ac35/

Reference: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by FareesHussain**
## Explanation

Fixes #2118 
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

Portrait | Landscape 
--- | --- 
 |

### CODE REVIEW

File: .idea/gradle.xml [hunk 1]
```diff
@@ -20,7 +20,8 @@
           </set>
         </option>
         <option name="resolveModulePerSourceSet" value="false" />
+        <option name="useQualifiedModuleNames" value="true" />
```
Comment (anandwana001) [inline .idea/gradle.xml]: please don't change anything in this file.

**Review by anandwana001 — CHANGES_REQUESTED** [review]
first pass, Thanks @FareesHussain

File: .idea/gradle.xml (hunk 1 above)
Comment (FareesHussain) [reply .idea/gradle.xml]: ok, I accidentally pushed

Comment (anandwana001) [reply .idea/gradle.xml]: @FareesHussain to make sure the suggested changes are done, the author of the review requested will be the one who resolve these comments. What best the PR author can do is make some comments like done if it is done or anything if needs a conversation ahead. Please don't resolve such comments by yourself.

Comment (FareesHussain) [reply .idea/gradle.xml]: ok

**Comment by rt4914** [comment]
@FareesHussain As per the second point in this https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change
Please make sure that your branch has all small characters separated by `-`. Something like `admin-pin-activity-tablet-ui`. You won't be able to do this now but please follow this convention in future PRs.

File: app/src/main/res/layout-sw600dp-land/admin_pin_activity.xml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      android:clipToPadding="false"
+      android:paddingStart="232dp"
+      android:paddingEnd="232dp"
+      android:paddingBottom="108dp">
+
+      <View
+        android:layout_width="match_parent"
+        android:layout_height="6dp"
+        android:layout_marginStart="-232dp"
+        android:layout_marginEnd="-232dp"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/admin_pin_activity.xml]: Never mention padding/margin in negative.

File: app/src/main/res/layout-sw600dp-land/admin_pin_activity.xml [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    android:fillViewport="true">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+      android:id="@+id/admin_auth_container"
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:background="@color/oppiaBackgroundYellowIvory"
+      android:clipToPadding="false"
+      android:paddingStart="232dp"
+      android:paddingEnd="232dp"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/admin_pin_activity.xml]: Similar to this comment, you should apply the concepts here too. https://github.com/oppia/oppia-android/pull/2124#pullrequestreview-533595345

**Review by rt4914 — CHANGES_REQUESTED** [review]
PTAL @FareesHussain

File: app/src/main/res/layout-sw600dp-land/admin_pin_activity.xml (hunk 3 above)
Comment (FareesHussain) [reply app/src/main/res/layout-sw600dp-land/admin_pin_activity.xml:26]: do I have to reformat it?

File: app/src/main/res/layout-sw600dp-land/admin_pin_activity.xml (hunk 2 above)
Comment (FareesHussain) [reply app/src/main/res/layout-sw600dp-land/admin_pin_activity.xml]: Done, please check

File: app/src/main/res/layout-sw600dp-land/admin_pin_activity.xml (hunk 3 above)
Comment (FareesHussain) [reply app/src/main/res/layout-sw600dp-land/admin_pin_activity.xml]: @rt4914 can you specify which concepts
is it negative margin/padding or reformatting code

**Comment by FareesHussain** [comment]
@rt4914 please check

File: app/src/main/res/layout-sw600dp-land/admin_pin_activity.xml (hunk 3 above)
Comment (FareesHussain) [reply app/src/main/res/layout-sw600dp-land/admin_pin_activity.xml]: Done

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @FareesHussain

**Comment by rt4914** [comment]
@FareesHussain Can you please update the PR similar to what we just did in https://github.com/oppia/oppia-android/pull/2124

Mainly focusing on `minWidth=360dp` and adjusting internal UI accordingly.

**Comment by FareesHussain** [comment]
Okay

**Comment by FareesHussain** [comment]
@rt4914 Done, Please check.

**Comment by anandwana001** [comment]
Can we have test cases related to these layouts too? @rt4914

**Comment by rt4914** [comment]
> Can we have test cases related to these layouts too? @rt4914

@anandwana001 What type of tests do we need in this?

**Comment by anandwana001** [comment]
> > Can we have test cases related to these layouts too? @rt4914
> 
> @anandwana001 What type of tests do we need in this?

@rt4914 
As we are introducing the tablet screens, all the pre written tests should pass for this UI as well.

**Comment by FareesHussain** [comment]
> As we are introducing the tablet screens, all the pre written tests should pass for this UI as well.

@anandwana001 I tried to run AdminPinActivityTest but It takes too long even for mobiles maybe there's an issue

**Comment by anandwana001** [comment]
> > As we are introducing the tablet screens, all the pre written tests should pass for this UI as well.
> 
> @anandwana001 I tried to run AdminPinActivityTest but It takes too long even for mobiles maybe there's an issue

Did you run on emulator?

**Comment by FareesHussain** [comment]
> Did you run on emulator?

Yes

**Comment by anandwana001** [comment]
> Yes

All the test cases should run correctly, as they are working correctly on develop branch.

**Comment by rt4914** [comment]
> > > Can we have test cases related to these layouts too? @rt4914
> > 
> > 
> > @anandwana001 What type of tests do we need in this?
> 
> @rt4914
> As we are introducing the tablet screens, all the pre written tests should pass for this UI as well.

@anandwana001 But I don't see any special case like difference in implementation or span changes that's why I think these tests should pass on tablet too.

**Comment by FareesHussain** [comment]
> @anandwana001 But I don't see any special case like difference in implementation or span changes that's why I think these tests should pass on tablet too.

Yes as long all view ids are the same there won't be any problem with tests (as far as I know)

File: .idea/gradle.xml (hunk 1 above)
Comment (rt4914) [reply .idea/gradle.xml]: Resolving this on behalf of @anandwana001

**Comment by anandwana001** [comment]
> > > > Can we have test cases related to these layouts too? @rt4914
> > > 
> > > 
> > > @anandwana001 What type of tests do we need in this?
> > 
> > 
> > @rt4914
> > As we are introducing the tablet screens, all the pre written tests should pass for this UI as well.
> 
> @anandwana001 But I don't see any special case like difference in implementation or span changes that's why I think these tests should pass on tablet too.

yes, just for the reference, it's better to run and check if all tests are passing on tablet device as we don't have espresso check in our github action.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**